webapp/packages/plugin-navigation-tree/src/TreeNew/useTreeData.ts

```diff
@@ -27,27 +27,33 @@ interface IOptions {
   getChildren: (node: string) => string[];
   load(nodeId: string, manual: boolean): Promise<void>;
 
+  getParent?: (node: string) => string | null;
   childrenTransformers?: TreeDataTransformer<string[]>[];
   nodeTransformers?: TreeDataTransformer<INode>[];
+  parentTransformers?: TreeDataTransformer<string | null>[];
   stateTransformers?: TreeDataTransformer<INodeState>[];
 }
 
 export function useTreeData(options: IOptions): ITreeData {
   options = useObservableRef(
     {
       ...options,
+      getParent: options.getParent ?? (() => null),
```

Comment:
you don't need to fallback to `() => null` because the type remains nullable and later, you still using ? operand `options.getParent?.(id)`